src/pattern-library/components/Library.js

```diff
@@ -151,40 +196,40 @@ function Demo({ children, withSource = false, style = {}, title }) {
     );
   });
   return (
-    <div>
-      {title && (
-        <div className="py-4">
-          <h4>{title}</h4>
+    <div className="space-y-2 p-4">
+      <div className="flex items-center">
+        <div className="py-2 grow">
+          <h4 className="text-lg italic text-slate-7 font-light">{title}</h4>
+        </div>
+        <div className="flex flex-row items-center justify-end gap-x-4">
+          {withSource && (
+            <>
+              <DemoButton
+                onClick={() => setVisibleTab('demo')}
+                pressed={visibleTab === 'demo'}
+              >
+                Demo
+              </DemoButton>
```

Comment:
If `withSource` is false, we don't render the "Demo" OR "Source" buttons. I assume this is because there is only one tab in that case, so it is redundant. Previously a "Demo" tab title was rendered in both cases.

Reply:
Correct. This is meant to be an enhancement.